hedera-node/hedera-schedule-service-impl/src/test/java/com/hedera/node/app/service/schedule/impl/test/handlers/ScheduleDeleteHandlerTest.java

```diff
@@ -16,115 +16,72 @@
-import com.hedera.node.app.spi.state.ReadableKVStateBase;
+import com.hedera.node.app.spi.fixtures.Assertions;
 import com.hedera.node.app.spi.workflows.PreCheckException;
+import com.hedera.node.app.spi.workflows.PreHandleContext;
+import com.hedera.node.app.workflows.prehandle.PreHandleContextImpl;
 import java.security.InvalidKeyException;
-import java.util.Optional;
 import java.util.Set;
+import org.assertj.core.api.BDDAssertions;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.mockito.BDDMockito;
-import org.mockito.Mock;
 
 class ScheduleDeleteHandlerTest extends ScheduleHandlerTestBase {
-    private final ScheduleID scheduleID =
-            ScheduleID.newBuilder().scheduleNum(100L).build();
     private final AccountID scheduleDeleter =
             AccountID.newBuilder().accountNum(3001L).build();
-    private final ScheduleDeleteHandler subject = new ScheduleDeleteHandler();
-    protected TransactionBody scheduledTxn;
-
-    @Mock
-    private ScheduleVirtualValue schedule;
-
-    @Mock
-    private ReadableKVStateBase<Long, ScheduleVirtualValue> schedulesById;
 
-    private ReadableScheduleStore scheduleStore;
+    private ScheduleDeleteHandler subject;
+    private PreHandleContext realPreContext;
 
     @BeforeEach
-    void setUp() {
-        BDDMockito.given(states.<Long, ScheduleVirtualValue>get("SCHEDULES_BY_ID"))
-                .willReturn(schedulesById);
-        scheduleStore = new ReadableScheduleStoreImpl(states);
+    void setUp() throws Exception {
+        setUpBase();
+        subject = new ScheduleDeleteHandler();
+        BDDMockito.given(accountStore.getAccountById(scheduleDeleter)).willReturn(payerAccount);
     }
 
     @Test
-    void scheduleDeleteHappyPath() throws InvalidKeyException, PreCheckException, InvalidKeyException {
-        final var txn = scheduleDeleteTransaction();
-        scheduledTxn = givenSetupForScheduleDelete(txn);
-        BDDMockito.given(schedule.hasAdminKey()).willReturn(true);
-        BDDMockito.given(schedule.adminKey()).willReturn(Optional.of(JKey.mapKey(TEST_KEY)));
-        BDDMockito.given(schedulesById.get(scheduleID.scheduleNum())).willReturn(schedule);
+    void scheduleDeleteHappyPath() throws InvalidKeyException, PreCheckException {
+        realPreContext =
+                new PreHandleContextImpl(mockStoreFactory, scheduleDeleteTransaction(), testConfig, mockDispatcher);
 
-        final var context = new FakePreHandleContext(accountStore, txn);
-        context.registerStore(ReadableScheduleStore.class, scheduleStore);
-
-        subject.preHandle(context);
-        assertEquals(scheduleDeleter, context.payer());
-        assertEquals(Set.of(), context.requiredNonPayerKeys());
+        subject.preHandle(realPreContext);
+        BDDAssertions.assertThat(scheduleDeleter).isEqualTo(realPreContext.payer());
+        BDDAssertions.assertThat(Set.of()).isNotEqualTo(realPreContext.requiredNonPayerKeys());
```

Comment:
Two comments:

Format is typically `assertThat(<test result>).isEqualTo(<expected value>)`

Also, wouldn't this assertion be much more useful if compared with the expected set instead of just making sure it's not empty?

Reply:
Improving these is a future PR, which will clean up a lot of these rough assertions.